cmp/zoneGrid/impl/ZoneGridPersistenceModel.ts

```diff
@@ -22,7 +23,14 @@ export class ZoneGridPersistenceModel extends HoistModel {
     zoneGridModel: ZoneGridModel;
 
     @observable.ref
-    state: PlainObject;
+    state: {
+        columns?: Partial<ColumnState>[];
+        sortBy?: GridSorterLike;
+        groupBy?: Some<string>;
+        version?: number;
+        autosize?: any;
+        mappings?: Record<Zone, Some<string | ZoneMapping>>;
+    };
```

Comment:
I don't believe we save `columns` or `autosize` in this state block - N/A / replaced by `mappings`

Reply:
good catch